packages/core/src/view/handler/CellEditorHandler.ts

```diff
@@ -799,7 +799,7 @@ class CellEditorHandler implements GraphPlugin {
       if (
         this.autoSize &&
         // @ts-ignore
```

Comment:
**thought:** the ts-ignore directive should not be necessary if the code is correctly fixed.
I am checking this and will push a change directly if this is now correct.

Reply:
Done in 5de66571